src/main/java/create_task_use_case/CreateTaskInputData.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Create Task Input Data Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 27, 2022
+ */
+package create_task_use_case;
+
+public class CreateTaskInputData {
+
+    private String title;
+    private int dayIndex;
+
+    public CreateTaskInputData(String title, String dayIndex){
+        this.title = title;
+        this.dayIndex = 0;
```

Comment:
I believe 
this.dayIndex = 0; 
is unnecessary and redundant.

Reply:
Apologizes, missed that while refactoring